### PR TITLE
Use Branch and BranchMut for get and get_mut methods

### PR DESCRIPTION
**Pull request by krl**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dusk-kelvin-map"
-version = "0.1.1"
+version = "0.1.2"
```
Comment (vlopes11) [inline Cargo.toml]: It is a breaking change, so the new version should be `0.2.0`

**Review by vlopes11 — DISMISSED** [review]
LGTM, just fix the version num

**Comment by krl** [comment]
fixes #8

**Review by vlopes11 — APPROVED** [review]
LGTM

**Pull request merged**